src/test/java/Homework19.java

```diff
@@ -0,0 +1,27 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+
+public class Homework19 extends BaseTest {
+    @Test
+    public void deletePlaylist() throws InterruptedException{
+        login("ryan.carter@testpro.io", "te$t$tudent");
+        playlist();
+        verifyPlaylistDeleted();
+        Thread.sleep(2000);
+    }
+    public void playlist() throws InterruptedException {
+        WebElement songs = driver.findElement(By.cssSelector("#playlists > ul > li:nth-child(5) > a"));
```

Comment:
This locator is bad.
Try to specify locator of an exact element you want to find (example: you can find it by inner text)
Little hint (XPath) - //li[@class="playlist playlist"]/a[text() = "putYourPlaylistNameHere"] - (There is even shorter locator you can write)